internal/env/cert.go

```diff
@@ -22,6 +25,25 @@ func Domain() string {
 	return domain
 }
 
+// WhiteList returns a list of domains that the server will create certificates for.
+func WhiteList() ([]string, error) {
+	domains := os.Getenv("QUICKFEED_WHITELIST")
+	if domains == "" {
+		return []string{}, errors.New("no domains in whitelist. Please set the WHITELIST environment variable")
```

Comment:
Maybe avoid mentioning the env variable to avoid forgetting to update both ;-) 

What about:
```
required whitelist is undefined.
```
Also, you should `return nil, errors...` instead of constructing an empty slice. Since if a user of `Whitelist()` forgets to check for `nil` error, the empty slice returned might cause a problem... (probably not, but better to be safe...)